db/seeds.rb

```diff
@@ -5,3 +5,108 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# Default admin
+
+User.create(
+  email: "admin@subway-reviews.com",
+  password: "password",
+  password_confirmation: "password",
+  role: "admin"
+  )
+
+lines = Line.create([
+  {name: "Red"},
+  {name: "Green"},
+  {name: "Blue"},
+  {name: "Orange"},
+  ])
+
+require 'open-uri'
+require 'nokogiri'
+require 'json'
+
+green_line_stops =[]
+red_line_stops =[]
+orange_line_stops =[]
+blue_line_stops =[]
+silver_line_stops =[]
+
+def grab_stop_names(route_ids, color_line_stops)
+  route_ids.each do |route_id|
+    routes_by_stop = open("http://realtime.mbta.com/developer/api/v2/stopsbyroute?api_key=wX9NwuHnZU2ToO7GmGR9uw&route=#{route_id}&format=json")
+    route_by_stop_data = JSON.load(routes_by_stop)
+    route_by_stop_data["direction"][0]["stop"].each {|stop|
+      color_line_stops << {name: stop["parent_station_name"], latitude: stop["stop_lat"], longitude: stop["stop_lon"]}}
```

Comment:
Line is too long. [119/80]<br>Expression at 40, 119 should be on its own line.<br>Space inside { missing.<br>Space inside } missing.